contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -356,4 +443,26 @@ contract WithdrawalQueue is AccessControlEnumerable, WithdrawalQueueBase {
 
         return _recipient;
     }
+
+    modifier onlyValidWithdrawalRequestInput(WithdrawalRequestInput memory _withdrawalRequestInput) {
+        address token = _withdrawalRequestInput.token;
+        if (token != address(STETH) && token != address(WSTETH)) {
+            address[] memory withdrawableTokens = new address[](2);
```

Comment:
maybe too sophisticated error handling

Reply:
Fixed. This check and the modifier were removed due to the methods interface change.